### PR TITLE
Expanding description of viewing Product Reviews via Magento Admin

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) expands description of viewing Product Reviews via Magento Admin

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/marketing/magento-product-reviews.html

### CODE REVIEW

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you for reviewing ☀️
I have implemented your suggestions, could you please approve?
Thank you!

File: src/marketing/magento-product-reviews.md [hunk 1]
```diff
@@ -19,3 +19,18 @@ The number of stars indicates the product's satisfaction rating. Visitors can cl
...
 ![Example storefront - Reviews tab]({% link images/images/storefront-reviews-tab.png %}){: .zoom}
 _Reviews Tab_
+
+## View Product Review via Magento Admin
+
+For viewing all reviews for a specific product via Magento Admin do the following:
+
+1. On the _Admin_ sidebar, go to **Catalog** > **Products**.
+
+1. Find the products that you want to view and click **Edit** in the _Action_ column.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/marketing/magento-product-reviews.md]: Done ✅

File: src/marketing/magento-product-reviews.md [hunk 2]
```diff
@@ -19,3 +19,18 @@ The number of stars indicates the product's satisfaction rating. Visitors can cl
 
 ![Example storefront - Reviews tab]({% link images/images/storefront-reviews-tab.png %}){: .zoom}
 _Reviews Tab_
+
+## View Product Review via Magento Admin
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/marketing/magento-product-reviews.md]: Done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**